Makefile

```diff
@@ -620,9 +630,13 @@ libroachccl: $(LIBROACH_DIR)/Makefile libroach
 	@$(MAKE) --no-print-directory -C $(LIBROACH_DIR) roachccl
 
 PHONY: check-libroach
-check-libroach: $(LIBROACH_DIR)/Makefile
+check-libroach: $(LIBROACH_DIR)/Makefile libgtest
```

Comment:
By my read, this `libgtest` is built into `~/go/native/TRIPLE/googletest` and is actually entirely ignored by libroach's CMake, which builds its own version at `~/go/native/TRIPLE/libroach/googletest` thanks to the magic of `add_subdirectory`.

Reply:
See main comment: our cmake doesn't build `googletest`, it just needs a built library it can link in. This is what `add_subdirectory` does.